paper-shadow.html

```diff
@@ -23,34 +23,90 @@
 -->
 
 <link href="../polymer/polymer.html" rel="import">
+<link href="../layout/layout.html" rel="import">
+<dom-module id="paper-shadow">
+  <style>
+    paper-shadow {
+      display: block;
+      position: relative;
+      pointer-events: none;
+    }
+
+    paper-shadow #shadow-bottom,
+    paper-shadow #shadow-top {
+      border-radius: inherit;
+    }
+
+    paper-shadow #shadow-bottom[animated],
+    paper-shadow #shadow-top[animated] {
+      transition: box-shadow 0.28s cubic-bezier(0.4, 0, 0.2, 1);
+    }
+
+    paper-shadow #shadow-top.paper-shadow-z-1 {
+      box-shadow: none;
+    }
+
+    paper-shadow #shadow-bottom.paper-shadow-z-1 {
+      box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.37);
+    }
```

Comment:
@cdata I can't remember why and I can't think of a good reason why I preferred to keep the styled element in the local DOM, so I vote for simplifying.

@ssorallen The issue I was referring to is when a user binds to paper-shadow's class attribute, like so

```
<paper-shadow class="{{myClassList}}"></paper-shadow>
```

then the classes applied by paper-shadow will be overwritten. Yes, you can still modify the attributes on host but perhaps it's more obvious that they're private to the host and modifies its behavior (in this element's case, the styling). For example, 

```html
<style>
  paper-shadow[z=1] {
    box-shadow: /* something */
  }
</style>
<paper-shadow z=1></paper-shadow>
```

is pretty clear that the element wants to do something internally with the z attribute.